README.md

```diff
@@ -1,76 +1,231 @@
+        <li><a href="#having-problems">Having problems?</a></li>
+    </ul>
 </ul>
-<p align="center">
-	<img src="https://github.com/Zagrios/bs-manager/blob/master/resources/readme/beat-conflict.png?raw=true" width="130"/>
-</p>
 
+<!-- ABOUT THE PROJECT -->
+<div align="center">
+    <h1> </h1>
+    <h1><b>About The Project</b></h1>
+</div>
+<div>
+    <a href="https://github.com/Zagrios/bs-manager">BSManager</a> was created to unify and improve the two most used tools in the community (<a href="https://github.com/RiskiVR/BSLegacyLauncher">BS legacy launcher</a> and <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>).
+    <br><br>
+    <h4><a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to:</h4>
+    <ul>
+        <li><b>Manage instances of multiple versions</b>, designed for people who need to play on different versions or have multiple profiles for the same version.</li>
+        <li><b>Download your maps in the same way</b>, in addition to that, the maps are not version dependent, so you will be able to find your maps whatever version you are using without having to transfer them.</li>
+        <li><b>Install your mods in one click</b>, just like <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>, <a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to install your mods quickly and securely depending on the version you choose.</li>
+        <li><b>Customize your interface</b>, we know that some of you like to customize your interfaces to the maximum. That's why <a href="https://github.com/Zagrios/bs-manager">BSManager</a> brings you peace of mind on this point, you can change the name of the versions, their color, and you can even change the color of the logo! Of course, a dark theme has been added to protect your eyes at night.</li>
+    </ul>
+</div>
+<br>
+<div align="center">
+    <img src="resources/readme/preview.png" alt="preview">
+</div>
+<br>
+
+<!--BUILT WITH-->
+<div>
+    <h2><b>Built With</b></h2>
+    <ul>
+        <li>
+            <a href="https://electron-react-boilerplate.js.org/"><img src="https://img.shields.io/badge/-Electron%20React%20Boilerplate-black?style=for-the-badge&logo=Electron" alt="Electron React Boilerplate"></a>
+        </li>
+        <li>
+            <a href="https://github.com/SteamRE/DepotDownloader"><img src="https://img.shields.io/badge/-Depot%20Downloader-2a475e?style=for-the-badge&logo=steam" alt="Depot Downloader"></a>
+        </li>
+        <li>
+            <a href="https://rxjs.dev/"><img src="https://img.shields.io/badge/-RxJs-purple?style=for-the-badge&logo=ReactiveX" alt="RxJs"></a>
+        </li>
+        <li>
+            <a href="https://tailwindcss.com/"><img src="https://img.shields.io/badge/-Tailwind%20CSS-white?style=for-the-badge&logo=Tailwind%20CSS" alt="TailWind CSS"></a>
+        </li>
+    </ul>
+</div>
+<br>
+
+<!--CONTRIBUTING-->
+<div>
+    <h2><b>Contributing</b></h2>
```

Comment:
Contribution, Makers, Owners, i dont know but contributing is stange

Reply:
```suggestion
    <h2><b>Contribution</b></h2>
```